docs/user-guide/deployments.md

```diff
@@ -425,15 +425,27 @@ The `.spec.template` is a [pod template](/docs/user-guide/replication-controller
 the same schema as a [Pod](/docs/user-guide/pods), except it is nested and does not have an
 `apiVersion` or `kind`.
 
+In addition to required fields for a Pod, a pod template in a Deployment must specify appropriate
```

Comment:
regarding "appropriate"

Do we explicitly say anywhere that the set of labels selected must be unique within a namespace?